### DISCUSSION TITLE
Documents related to university certification

### DISCUSSION THREAD

**Discussion by Cho-Min-Su** [community/community#51514] [GitHub Education] [answered]
Hi there,

I'm currently in the process of university certification to get a student developer pack,
but I just don't understand why the documents I submitted can't be certified.

Currently, the documents already submitted have been returned, so the same documents cannot be submitted.

I'm attaching the document, so if anyone knows why it's not being certified, please kindly let me know..

Best Regards,
Minsu

Moderator's edit: I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Comment by devilshezi0** [comment]
components/search/SearchResults.tsx

**Comment by tusharhqq** [comment, marked as answer]
I think GitHub developers pack is for active students and as your certificate says that you have graduated in 2021, so i think you are no longer student

copied from github's official website
Make sure you pass the following requirements: 

Be currently enrolled in a degree or diploma-granting course of studies such as a high school, secondary school, college, university, homeschool, or similar educational institution.
Have a verifiable school-issued email address or upload documents that prove your current student status.
Have a GitHub user account.
Be at least 13 years old.
Documents that prove your current student status include a picture of your school ID, class schedule, transcript, and affiliation or enrollment verification letter.
During your tenure as a student, you may be prompted to periodically re-verify your current academic status.
